### DISCUSSION TITLE
Which model and config is the fastest on Deepgram?

### DISCUSSION THREAD

**Discussion by abbotadmin** [deepgram/community#835] [General help] [answered]
I'm just starting out and its important that I am able to optimize for speed for the audio files of about 1.5 mins length. I'm currently testing Nova 2 with smart format and punctuate turned on. What other options and models should I be looking at?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @abbotadmin, Nova-2 is our best blend of accuracy and speed. Smart formatting and punctuation do not increase the processing time. (Features that do raise latency include diarization and keywords.) 

If you want the absolute fastest model, and can accept slightly higher error rates, our original Base model is the lowest latency (`model=base`). 

You can check out our [Models & Languages doc](https://developers.deepgram.com/docs/models-languages-overview) as well.

**Reply by abbotadmin** [reply]
Thanks for the information. Unfortunately I can get much faster results on my nv 3080 locally using Whisper. I'm sure the accuracy isn't as great but it's suitable for now.
It's a shame Nova 2 just isn't up to scratch in terms of speed.
